nominees/fao-digital-services-portfolio-wapor.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "FAO Digital Services Portfolio - WAPOR",
+  "aliases": [
+    ""
```

Comment:
@nathanbaleeta - Should we add an alias here, similar to the entry for FAMEWS?

Reply:
Resolved in commit d1c3aad